util/src/servers/configuration/serviceinfo_api_server.rs

```diff
@@ -19,6 +19,7 @@ pub struct ServiceInfoApiServerSettings {
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(deny_unknown_fields)]
```

Comment:
We can delete that now that we know the correct format

Reply:
I think its better to add incase there is any wrong field/typo during creating .yml file , this macro will show error that some unknown field in the .yml, easy for debugging.

Reply:
Checked the docs, all good, I misunderstood :+1: